node_modules/oae-admin/manageuser/manageuser.html

```diff
@@ -192,6 +192,16 @@ <h3 id="manageuser-editprofile-header-container"><!-- --></h3>
                         </div>
                     </label>
                 </div>
+                <div class="col-sm-3 text-center">
+                    <label for="manageuser-email-never" {if userProfile.emailPreference === 'never'} class="checked"{/if}>
+                        <i class="fa fa-check hide"></i>
+                        <div class="oae-large-options well">
+                            <input type="radio" id="manageuser-email-never" value="never" name="manageuser-email-group" class="pull-left" tabindex="0" {if userProfile.emailPreference === 'never'} checked{/if}/>
+                            <i class="fa fa-ban large text-center {if userProfile.emailPreference === 'weekly'} selected{/if}"></i>
```

Comment:
No difference. The only rule that applies to `selected` (setting the color) is also defined to apply when the corresponding `<input>` is checked. In `skin.css`:

```
.fa.active, /* ... */  .fa.selected, /* ... */ input[type=radio]:checked+.fa {
    color: #555
}
```

Looks like the only other place it's used is in the preferences widget. I'll remove it there as well.